### ISSUE TITLE
Work tab scale issue

### ISSUE THREAD

**Issue by senyaak** [skyarkhangel/Hardcore-SK#1368]
When I use the interface scale above x1.25, the labels for jobs are missplaced.
When I expand tasks - the panel looks very bad, the tasks are placed on the top of each other and you just can't use it. See screenshots for more info. 
http://rgho.st/7yBTl5FgM 
http://rgho.st/865f6SJvF

**Comment by sidfu** [comment]
not a bug or issue. it is how it is when u expand all the jobs. u have to expand one at a time. also rimworld hates when u scale the ui. its long been known ui can bug when u scale it

**Comment by Alkanna** [comment]
I have to agree on this, would be really cool to be able to scroll left and right on the expanded work tab.

**Comment by Rhyssia** [comment]
Worked fine in A16 I believe it was, Changes to A17 kind of broke them. You can Ctrl+click to expand jobs individually and it looks a little better, but not much.

**Comment by mora145** [comment]
At the time it is a problem ... when there are many settlers and they are required only for specific jobs.

**Comment by sidfu** [comment]
well the base game itself has ui scalling issues. also 1.0 is suppose to change the ui.
dont forget if u hover over a block it does tell u the job in tooltip. thats what i use to find the job when i expand 1 job at time
